lib/docs/src/tab.stories.tsx

```diff
@@ -1,55 +1,101 @@
 import { Meta } from "@storybook/react";
-import { useEffect, useState } from "react";
+import { useState } from "react";
 import { DivPx, Switcher, Tab, Tabs } from "../../core/src";
+import { Utils } from "./utils";
+import {
+	GalleryTabDefault,
+	GalleryTabFlat,
+	GalleryTabHeight,
+} from "../../gallery/src/tab/tab";
 
-export default {
-	title: "Draft/Tabs",
+const meta: Meta = {
+	title: "Components/Tabs",
 	component: Tabs,
-} as Meta;
+	argTypes: {
+		noPadding: Utils.arg("boolean", "Visual"),
+		fullHeight: Utils.arg("boolean", "Visual"),
+		style: Utils.arg(Object.keys(Tabs.styles), "Visual"),
+		children: Utils.arg(null, "Controlled"),
```

Comment:
"children" is actually in both "controlled" and "uncontrolled" mode. It should be in another section, or just put it in "Visual" as well